docs/src/tutorials/software-management.md

```diff
@@ -103,7 +74,7 @@ Hover over the software entry which you want to remove and click the **Remove**
 
 ## Starting and Stopping the Software management services manually
```

Comment:
I would expect this in the howto.

Reply:
What would be useful here would be a hint on how to find on the device the cause of an error reported on the cloud.

Unfortunately, the UX is currently not good here.

Reply:
I agree with the first part. So, what to mention here?